### DISCUSSION TITLE
Passthrough object detection

### DISCUSSION THREAD

**Discussion by sabanadesan** [StereoKit/StereoKit#626] [Q&A]
Can you show some example code on object detection using Passthrough?

**Comment by maluoi** [comment]
Passthrough video is generally not accessible to developers, so you're likely looking more for something like the webcam! HoloLens 2 I know has a front-facing webcam used for this purpose. Additionally, the OpenXR spec doesn't have any object detection extensions that I know of, so StereoKit doesn't do this as part of its core functionality yet.

I don't have any examples on hand for doing object detection via webcam, but I'd recommend searching on those keywords for resources! I imagine there should be something good around for that in some shape or form.
